infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/AuthorizationManager.java

```diff
@@ -16,22 +16,18 @@
 import static tech.pegasys.teku.infrastructure.http.HttpStatusCodes.SC_UNAUTHORIZED;
 
 import io.javalin.http.Context;
-import io.javalin.http.Handler;
-import io.javalin.security.AccessManager;
-import io.javalin.security.RouteRole;
 import java.io.File;
 import java.io.IOException;
 import java.net.URLEncoder;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.Set;
 import java.util.stream.Stream;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import tech.pegasys.teku.infrastructure.exceptions.InvalidConfigurationException;
 
-public class AuthorizationManager implements AccessManager {
```

Comment:
I think it's cleaner to create `AuthorizationHandler` which implements `Handler` such as https://gist.github.com/StefanBratanov/2906e76016e59fe59da923df9a1e316d

Then can do `app.beforeMatched(new AuthorizationHandler...)`

Reply:
can do, i was just fixing the pattern we had... clearly its not breaking error messages this way... or we've missed a ton of testing...

Reply:
the context was still being set previously which is what does the error codes etc... even with that gist suggested there substantial changes for anything resembling our tests... just fyi - it wont look super similar.

Reply:
fixed now :)